hphp/runtime/vm/jit/code-cache.cpp

```diff
@@ -97,7 +98,14 @@ CodeCache::CodeCache()
     exit(1);
```

Comment:
That code quits if the code size is too small.

More generally, I don't think we should be altering these defaults on ARM, since your choice of platform isn't going to alter the reality of your codebase's size.  We should be looking for ways to optimize in spite of the direct jump size restrictions.